### ISSUE TITLE
Review propagation of Fastapi errors to Svelte endpoints

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-web#260]
TL;DR All Svelte endpoints must return the same response as the Fastapi one 
This holds for the moment, where Svelte endpoints are almost-identical copies of the Fastapi ones.

If we introduce more complex Svelte endpoints, then we can also return enriched responses (e.g. with additional information, or corresponding to an error which is specific to the Svelte endpoint).

**Comment by tcompa** [comment]
68ec8b49a2ee889c02700ddd0f3db7ea199ca557 already introduces
```js

export async function responseError(response) {
	throw error(response.status, await response.json())
}
```
which should be always used to propagate a FastAPI response as the response of the Svelte endpoint.

**Comment by tcompa** [comment]
For the record, we will try with the last option listed in https://kit.svelte.dev/docs/faq#how-do-i-use-x-with-sveltekit-how-do-i-use-a-different-backend-api-server

> If rewrites aren't an option, you could alternatively add an [API route](https://kit.svelte.dev/docs/routing#server):
```js
// `src/routes/api/[...path]/+server.js`:

/** @type {import('./$types').RequestHandler} */
export function GET({ params, url }) {
  return fetch(`https://my-api-server.com/${params.path + url.search}`);
}
```
> (Note that you may also need to proxy POST/PATCH etc requests, and forward request.headers, depending on your needs.)

**Comment by tcompa** [comment]
Closed with #288
